src/main/java/redis/clients/jedis/CommandObjects.java

```diff
@@ -4424,22 +4427,22 @@ public final CommandObject<Object> tFunctionCallAsync(String library, String fun
   /**
    * Get the instance for JsonObjectMapper if not null, otherwise a new instance reference with
    * default implementation will be created and returned.
-   * <p>This process of checking whether or not
-   * the instance reference exists follows <a
-   * href="https://en.wikipedia.org/wiki/Double-checked_locking#Usage_in_Java"
-   * target="_blank">'double-checked lock optimization'</a> approach to reduce the overhead of
-   * acquiring a lock by testing the lock criteria (the "lock hint") before acquiring the lock.</p>
+   *
```

Comment:
Double check lock is back.
```suggestion
   * <p>This process of checking whether or not
   * the instance reference exists follows <a
   * href="https://en.wikipedia.org/wiki/Double-checked_locking#Usage_in_Java"
   * target="_blank">'double-checked lock optimization'</a> approach to reduce the overhead of
   * acquiring a lock by testing the lock criteria (the "lock hint") before acquiring the lock.</p>
```